src/box/iproto_features.c

```diff
@@ -89,4 +89,6 @@ iproto_features_init(void)
 	iproto_features_set(&IPROTO_CURRENT_FEATURES,
 			    IPROTO_FEATURE_FETCH_SNAPSHOT_CURSOR);
 #endif /* defined(ENABLE_FETCH_SNAPSHOT_CURSOR) */
+	iproto_features_set(&IPROTO_CURRENT_FEATURES,
+			    IPROTO_FEATURE_IS_SYNC);
```

Comment:
In the commit message:

> Title: Document iproto feature FETCH_SNAPSHOT_CURSOR

Should be `IS_SYNC` apparently.

Also, we should mention the fix in the release notes. Please add a changelog entry.

Reply:
Added changelog, fixed commit msg